tests/test_sanitze.py

```diff
@@ -18,11 +17,10 @@
 
 
 @pytest.mark.parametrize(
-    "inp", INVALID_TEST_CASE_ARGS, ids=INVALID_TEST_CASE_IDS
+    "abs_path", INVALID_TEST_CASE_ARGS, ids=INVALID_TEST_CASE_IDS
 )
-def test_sanitize_invalid(inp: str):
-    with pytest.raises(plug.PlugError):
-        _sanitize.sanitize_text(inp)
+def test_sanitize_invalid(abs_path: pathlib.Path):
+    assert _syntax.check_syntax(abs_path.read_text().split("\n"))
```

Comment:
This is the only place I can find that you've changed in regards to returning the absolute path instead of the text. And the path is not used for anything but reading the text. Why not do that in the helper function instead?

You've also missed specifying `utf8` as the encoding when reading the text, so this will crash on a system that uses a different default encoding.